pkg/tests/ossm/smoke_test.go

```diff
@@ -63,6 +126,36 @@ func TestBookinfoInjection(t *testing.T) {
 	})
 }
 
+func assertProxiesReadyInLessThan10Seconds(t TestHelper, ns string) {
+	t.Log("Extracting proxy startup time and last transition time for all the pods in the namespace")
+	startedAt := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.status.containerStatuses[?(@.name=="istio-proxy")].state.running.startedAt}{"\n"}{end}`)
+	readyAt := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.status.conditions[?(@.type=="Ready")].lastTransitionTime}{"\n"}{end}`)
+
+	startedAtList := strings.Split(startedAt, "\n")
+	readyAtList := strings.Split(readyAt, "\n")
+
+	for i := 0; i < len(startedAtList) && i < len(readyAtList); i++ {
+		startedAtLine := startedAtList[i]
+		readyAtLine := readyAtList[i]
+		if startedAtLine == "" || readyAtLine == "" {
+			continue
+		}
+
+		podStartedAt, err := time.Parse(time.RFC3339, startedAtLine)
+		if err != nil {
+			t.Fatalf("Error parsing time for pod %d: %s", i, err)
+		}
+		podReadyAt, err := time.Parse(time.RFC3339, readyAtLine)
+		if err != nil {
+			t.Fatalf("Error parsing time for pod %d: %s", i, err)
+		}
+		startupTime := podReadyAt.Sub(podStartedAt)
+		if startupTime > 10*time.Second {
```

Comment:
Actually, 10 seconds is fairly long for a test cluster. istio-proxy should start within ~1-2 seconds, so we might check whether proxies become ready in less than 3 seconds. What do you think?